### ISSUE TITLE
Output patch must contains utoc extension?

### ISSUE THREAD

**Issue by Varyag-Ericsson** [rm-NoobInCoding/UnrealReZen#48]
Hi :)

I keep getting this error when trying to pack the modified files. Any help?
Thanks!

**Comment by rm-NoobInCoding** [comment]
Take a look at the example in readme
C:\Path\To\Folder\MyFile.utoc

**Comment by Varyag-Ericsson** [comment]
Right, that's exactly what I did, and yet I got that error FTL: Output path must contains utoc extension :)

I'm trying to pack Lollipop Chainsaw UI localization files as a patch, they are inside .utoc.

So my code is:
UnrealReZen.exe --content-path C:\Program Files (x86)\Steam\steamapps\common\LOLLIPOP CHAINSAW RePOP\lollipop\Content\Paks\L10NuaUI --compression-format Zlib --engine-version GAME_UE5_3 --game-dir C:\Program Files (x86)\Steam\steamapps\common\LOLLIPOP CHAINSAW RePOP\lollipop\Content\Paks --output-path C:\Program Files (x86)\Steam\steamapps\common\LOLLIPOP CHAINSAW RePOP\lollipop\Content\Paks\TestL10NuaUI.utoc

**Comment by rm-NoobInCoding** [comment]
Check #39

**Comment by Varyag-Ericsson** [comment]
Hi @rm-NoobInCoding
Thanks, quotes helped to start the process, yet it's not patching/creating any new files. Any advice, please?
![Screensot1](https://github.com/user-attachments/assets/ce4d6c04-d695-4b52-99e7-afc9fa5d9d0f)

**Comment by rm-NoobInCoding** [comment]
Its a known issue #45 
I will trying to fix it in next week
